docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       cache_from:
         - batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
     image: batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
-    command: ash -c "rm -f tmp/pids/server.pid && bundle exec rails server -p 3000 -b '0.0.0.0'"
+    command: ash -c "rm -f tmp/pids/server.pid && bundle exec rails db:migrate && bundle exec rails server -p 3000 -b '0.0.0.0'"
```

Comment:
I think we need this to be part of `Dockerfile` so that it's run in the Azure app .
server.

Reply:
refer to https://github.com/DFE-Digital/manage-courses-backend/blob/05e32186b351c61c823382dc86775fa9d09ef8ea/.travis.yml#L12

`bundle exec rails db:migrate` may just blow up as, basically `db` sever may be up but `db database` server in side of `db` may not be up

Reply:
you are correct, sir. Doing that now...

Reply:
cheers sau